test/Microsoft.AspNetCore.SystemWebAdapters.Tests/VirtualPathUtilityTests.cs

```diff
@@ -0,0 +1,212 @@
+using System;
+using System.Web;
+using Xunit;
+
+namespace Microsoft.AspNetCore.SystemWebAdapters.Tests
+{
+    public class VirtualPathUtilityTests
+    {
+        [InlineData("/", "/")]
+        [InlineData("~/", "/")]
+        [Theory]
+        public void ToAbsolute(string virtualPath, string expected)
+        {
+            Assert.Equal(expected, (string?)VirtualPathUtility.ToAbsolute(virtualPath, "/"));
+        }
+
+        [Fact]
+        public void ToAbsoluteError()
+        {
+#if NET40_OR_GREATER
```

Comment:
I've generally not been running the tests on framework mostly because the mocking ends up being drastically different. 

These tests could potentially be more exhaustive by changing the implementation to be an (internal) instance class that takes in a virtual directory class and the static calls just go through a cached instance that has the result of HttpRuntime.AppVirtualDirectory

Reply:
I'm happy to take the conditionals out, but they were useful when I was running to compare a net48 project against the actual System.Web implementation.

Reply:
This is fine